kubernetes/resource_kubernetes_config_map_v1_data.go

```diff
@@ -0,0 +1,213 @@
+package kubernetes
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
`v1` and `metav1` seem to be a name of the same package that is imported twice and then below you use both names

Reply:
Yep you're right, did this by accident.